components/AdminTable/AdminDiscussionsTable/AdminDiscussionsTable.vue

```diff
@@ -21,18 +28,26 @@
         >
           {{ t("Last comment") }}
         </AdminTableTh>
+        <AdminTableTh
+          :sorted="sorted('created')"
+          scope="col"
+          class="w-44"
+          @sort="(direction: SortDirection) => $emit('sort', 'created', direction)"
+        >
+          {{ t("Created at") }}
+        </AdminTableTh>
         <AdminTableTh
           :sorted="sorted('closed')"
           scope="col"
+          class="w-44"
           @sort="(direction: SortDirection) => $emit('sort', 'closed', direction)"
         >
           {{ t("Closed at") }}
         </AdminTableTh>
         <AdminTableTh
           scope="col"
-          class="w-1/5"
         >
-          {{ t("Link to the discussion") }}
+          {{ t("Actions") }}
```

Comment:
Can we force a width to allow both buttons to be on the same line ?

Reply:
Done in 731839a